src/picongpu/include/plugins/EnergyFields.hpp

```diff
@@ -219,7 +219,7 @@ class EnergyFields : public ISimulationPlugin
         mpiReduce(nvidia::functors::Add(),
                   &globalFieldEnergy,
                   &localReducedFieldEnergy,
-                  2,
+                  1, /* only 1 reduce because type is float3_x[2] */
```

Comment:
after thinking about the fix I think we should fix it the other way around, this would be more intuitive

```
globalFieldEnergy,
localReducedFieldEnergy,
2,
mpi::reduceMethods::Reduce());
```